app/containers/HomeSettings/index.js

```diff
@@ -213,6 +239,39 @@ class SettingsPage extends Component<Props> {
     });
   }
 
+  renderPaths(paths, selectedPath) {
+    const { theme } = this.props;
+    return paths.map((path, index) => {
+      const label = path.get('label');
+      const derivation = path.get('derivation');
+      const selected = selectedPath === label;
+      const option = (
+        <SelectOption>
+          <OptionStatus>
```

Comment:
Please use the `selected` variable on `OptionStatus`

Reply:
Should I do the same changes to

const selected = selectedNode === name;
      const option = (
        <SelectOption>
          <OptionStatus>
            {selected ? (
              <Check

Reply:
No, there is no need for <Check>